README.md

```diff
@@ -39,12 +39,10 @@ dependency of Argo, so you don't need to specify it in your Cartfile.
 
 [CocoaPods]: http://cocoapods.org
 
-__DISCLAIMER: CocoaPods doesn't officially support Swift projects yet. Use the
```

Comment:
I wouldn't say it's that important. In a few days time the minimum version of CocoaPods permitted with the master specs repository will be bumped to 0.36. Due to numerous backwards incompatible changes in pod specs.

https://github.com/CocoaPods/Specs/blob/master/CocoaPods-version.yml#L2